SparseGrids/CMakeLists.txt

```diff
@@ -111,6 +111,9 @@ endif()
 target_link_libraries(Tasmanian_libsparsegrid  Tasmanian_dependencies)
 
 target_include_directories(Tasmanian_libsparsegrid PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/>)
+# the following are needed for Exotic quadrature only
+target_include_directories(Tasmanian_libsparsegrid PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../Addons>)
```

Comment:
Note that this is technically not a proper way to use CMake. The correct thing to do is to move `tasgrid` into a separate folder that gets includes after the others. Thus, we can keep the hierarchy of sub-modules, e.g., Sparse Grid then DREAM then the Addons.

Don't do that now, we will do it after the release is finalized.